orchagent/notifications.cpp

```diff
@@ -52,6 +52,18 @@ void on_twamp_session_event(uint32_t count, sai_twamp_session_event_notification
     // which causes concurrency access to the DB
 }
 
+void on_ha_set_event(uint32_t count, sai_ha_set_event_data_t *data)
+{
+    // don't use this event handler, because it runs by libsairedis in a separate thread
```

Comment:
This won't work since DPU uses ZMQ. Please check this PR https://github.com/sonic-net/sonic-swss/pull/3547 on what has to be done

Reply:
Hi Vivek, thanks for bringing it up. Can you share the doc? Per HA detailed design, there is no change mentioned for communication mode between syncd to orchagent.